evm/src/omni-bridge/contracts/OmniBridge.sol

```diff
@@ -283,6 +294,28 @@ contract OmniBridge is
         proxy.upgradeToAndCall(implementation, bytes(""));
     }
 
+    function denormalizeAmount(address tokenAddress, uint128 amount) internal view returns (uint128) {
+        uint8 originDecimals = tokenOriginDecimals[tokenAddress];
+        uint8 decimals = IERC20Metadata(tokenAddress).decimals();
+
+        if (originDecimals > decimals) {
+            return amount * uint128(10) ** (originDecimals - decimals);
```

Comment:
We don't need this if statement. If `originDecimals == decimals`, then it would return `amount * 10^0` which is `amount`